site/src/pages/components/selectmenu.mdx

```diff
@@ -0,0 +1,257 @@
+  }
+</style>
+<selectmenu class="my-custom-select">
+  <div slot="listbox">
+    <div popup behavior="listbox">
+      <option>Option 1</option>
+      <option>Option 2</option>
+      <option>Option 3</option>
+      <option>Option 4</option>
+      <option>Option 5</option>
+    </div>
+  </div>
+</selectmenu>
+```
+
+The `popup` attribute used in this example's `<div popup>` is from the proposal at [Popup API (Explainer)](https://open-ui.org/components/popup.research.explainer). The element with `behavior="listbox"` is required to be a `<div popup=popup>`. Applying `behavior="listbox"` tells the browser to apply select menu listbox behavior to that element: it will be opened when the `<selectmenu>`'s button is clicked, and the user can select `<option>`s inside it with mouse, arrow keys, and touch.
+
+The above code snippet results in the following style:
+
+<Image
+  src="selectmenu-replacing-listbox-part.png"
+  alt="The rendering of a selectmenu control with the above HTML and CSS code"
+/>
+
+### Extending the markup
+
+Not only can you replace the default parts with your own, you can also extend the control's markup by adding new elements. This can be useful to augment the listbox or button with extra information, or to add new functionality.
+
+Consider the following example:
+
+```html
+<style>
+  .my-custom-select [slot='button'] {
+    display: flex;
+    align-items: center;
+    gap: 1rem;
+  }
+  .my-custom-select button {
+    border: none;
+    margin: 0;
+    padding: 0;
+    width: 2rem;
+    height: 2rem;
+    border-radius: 50%;
+    display: grid;
+    place-content: center;
+  }
+  .my-custom-select button::before {
+    content: '\25BC';
+  }
+  .my-custom-select [popup] {
+    padding: 0;
+  }
+  .my-custom-select .section {
+    padding: 1rem 0 0;
+    background: radial-gradient(ellipse 60% 50px at center top, #000a 0%, transparent 130%);
+  }
+  .my-custom-select h3 {
+    margin: 0 0 1rem 0;
+    text-align: center;
+    color: white;
+  }
+  .my-custom-select option {
+    text-align: center;
+    padding: 0.5rem;
+  }
+</style>
+<selectmenu class="my-custom-select">
+  <div slot="button">
+    <span class="label">Choose a plant</span>
+    <span behavior="selected-value" slot="selected-value"></span>
+    <button behavior="button"></button>
+  </div>
+  <div slot="listbox">
+    <div popup behavior="listbox">
+      <div class="section">
+        <h3>Flowers</h3>
+        <option>Rose</option>
+        <option>Lily</option>
+        <option>Orchid</option>
+        <option>Tulip</option>
+      </div>
+      <div class="section">
+        <h3>Trees</h3>
+        <option>Weeping willow</option>
+        <option>Dragon tree</option>
+        <option>Giant sequoia</option>
+      </div>
+    </div>
+  </div>
+</selectmenu>
+```
+
+Using custom markup to wrap the list of options, the above example creates sections with their own styles and content as seen below:
+
+<Image
+  src="selectmenu-using-custom-markup.png"
```

Comment:
I've noticed that the other images use the following prefix before the file name "/images/". did you intentionally omit this?

Reply:
This was not intentional, I've added the absolute path